pkg/convert/po.go

```diff
@@ -245,9 +245,9 @@ func ToPo(t model.Translation) ([]byte, error) {
 			switch p := p.(type) {
 			case ast.TextPattern:
 				text += string(p)
-			case ast.PlaceholderPattern:
+			case ast.Pattern:
 				//nolint:forcetypeassert // No other types are possible.
-				text += placeholders[p.Expression.(ast.VariableExpression).Variable]
+				text += placeholders[p.(ast.VariableExpression).Variable]
```

Comment:
What's the point of asserting interface type to the interface? I am curious why golang allows doing that :)

```go
case ast.VariableExpression: // No other types are possible for now
  text += placeholders[p.Variable]
```